src/components/ElementBar/TextAlignmentButtons/TextAlignmentButtons.test.tsx

```diff
@@ -0,0 +1,143 @@
+  mockWhiteboardManager,
+} from '../../../lib/testUtils/documentTestUtils';
+import { WhiteboardManager } from '../../../state';
+import { Toolbar } from '../../common/Toolbar';
+import { TextAlignmentButtons } from './TextAlignmentButtons';
+
+let widgetApi: MockedWidgetApi;
+
+afterEach(() => widgetApi.stop());
+
+beforeEach(() => (widgetApi = mockWidgetApi()));
+
+describe('<TextAlignmentButtons/>', () => {
+  let whiteboardManager: jest.Mocked<WhiteboardManager>;
+  let Wrapper: ComponentType<PropsWithChildren<{}>>;
+
+  beforeEach(() => {
+    ({ whiteboardManager } = mockWhiteboardManager({
+      slides: [
+        [
+          'slide-0',
+          [
+            ['element-0', mockEllipseElement()],
+            ['element-1', mockLineElement()],
+          ],
+        ],
+      ],
+    }));
+    whiteboardManager
+      .getActiveWhiteboardInstance()
+      ?.getSlide('slide-0')
+      .setActiveElementId('element-0');
+
+    Wrapper = ({ children }) => (
+      <WhiteboardTestingContextProvider
+        whiteboardManager={whiteboardManager}
+        widgetApi={widgetApi}
+      >
+        <Toolbar>{children}</Toolbar>
+      </WhiteboardTestingContextProvider>
+    );
+  });
+
+  it('should render without exploding', async () => {
+    render(<TextAlignmentButtons />, { wrapper: Wrapper });
+
+    const radioGroup = screen.getByRole('radiogroup', {
+      name: 'Text Alignment',
+    });
+
+    expect(
+      within(radioGroup).getByRole('radio', { name: 'Left' }),
+    ).not.toBeChecked();
+    expect(
+      within(radioGroup).getByRole('radio', { name: 'Center' }),
+    ).toBeChecked();
+    expect(
+      within(radioGroup).getByRole('radio', { name: 'Right' }),
+    ).not.toBeChecked();
+  });
+
+  it('should have no accessibility violations', async () => {
+    const { container } = render(<TextAlignmentButtons />, {
+      wrapper: Wrapper,
+    });
+
+    expect(
+      screen.getByRole('radiogroup', { name: 'Text Alignment' }),
+    ).toBeInTheDocument();
+
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it('should show selected alignment', async () => {
+    whiteboardManager
+      .getActiveWhiteboardInstance()
+      ?.getSlide('slide-0')
+      .updateElement('element-0', { textAlignment: 'left' });
+
+    render(<TextAlignmentButtons />, { wrapper: Wrapper });
+
+    expect(screen.getByRole('radio', { name: 'Left' })).toBeChecked();
+    expect(screen.getByRole('radio', { name: 'Center' })).not.toBeChecked();
+    expect(screen.getByRole('radio', { name: 'Right' })).not.toBeChecked();
+  });
+
+  it('should hide for non-shape elements', async () => {
+    render(<TextAlignmentButtons />, { wrapper: Wrapper });
+
+    const radioGroup = screen.getByRole('radiogroup', {
+      name: 'Text Alignment',
+    });
+
+    act(() => {
+      whiteboardManager
+        .getActiveWhiteboardInstance()
+        ?.getSlide('slide-0')
+        .setActiveElementId('element-1');
+    });
+
+    await waitFor(() => {
```

Comment:
I like the idea. Although, I don't necessarily see it in this PR. Since, this is based on the work of a former colleague and is probably not the only place where we would need it. Therefore, it should be handled separately.